### PR TITLE
Add some high-level maintainership docs.

### PR DESCRIPTION
**Pull request by rfk**
I did a high-level pass over the docs here and tried to add things that might be useful, without going too overboard.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,4 +63,82 @@ Under the hood, the `PairingChannel` implements the "externally-provisioned
...
+
+Use `npm publish` to cut a new release, after ensuring that the tests pass and the
+correct artifacts have been built:
+
+```
+npm run build
+npm test
+npm publish
+git tag vX.Y.Z & git push origin vX.Y.Z
+```
```
Comment (rfk) [inline README.md:138]: I haven't actually tested this release process. We should do so once this batch of PRs land, and tweak as needed.

File: dist/FxAccountsPairingChannel.babel.umd.js [hunk 2]
```diff
@@ -7897,7 +7904,9 @@ if (
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 // A wrapper that combines a WebSocket to the channelserver
 // with some client-side encryption for securing the channel.
-// We'll improve the encryption before initial release...
```
Comment (rfk) [inline dist/FxAccountsPairingChannel.babel.umd.js:7900]: We actually did "improve the encryption" quite substantially before initial release 😅

File: README.md [hunk 3]
```diff
@@ -4,14 +4,23 @@ This repo implements a shared library for two javascript environments
...
 
 API
 ===
 
+To use this library, each side needs to agree on the URL of a
+[channelserver](https://github.com/mozilla-services/channelserver)
+instance which they can use to exchange messages via WebSocket:
+
+```
+const CHANNEL_SERVER_URL = "wss://dev.channelserver.nonprod.cloudops.mozgcp.net"
```
Comment (vbudhram) [inline README.md:21]: I haven't checked this in a while but I remember this not working and having to switch to https://github.com/mozilla/fxa/blob/e1fcec815dc52d81f1633528c43d434cd8dab058/packages/fxa-content-server/server/lib/configuration.js#L451 in content-server.

**Review by vbudhram — APPROVED** [review]
@rfk thanks for adding the documentation! LGTM 👍🏽

**Pull request merged**